### DISCUSSION TITLE
Allow Grafana to tolerate small windows where backend DB is inaccessible

### DISCUSSION THREAD

**Discussion by dansimone** [grafana/grafana#37316] [General]
Currently, when Grafana's backend Mysql DB is unavailable, Grafana crashes almost immediately, even when not being actively used.  To make Grafana more fault tolerant, some improvements to this behavior might be to:
- Have Grafana remain alive, but respond with an error message indicating a backend issue.
- Have Grafana run on cached data for some period of time before either crashing or showing an error message indicating a backend issue.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
